### PR TITLE
Add extras to loginoptions

### PR DESCRIPTION
**Pull request by maina-david**
This pull request extends the `LoginOptions` interface in the `@types/facebook-js-sdk` package to include the `extras` object. The `extras` object is a critical part of the Facebook JavaScript SDK's authentication process and configuration.

**Motivation and Background**

The `LoginOptions` interface is an integral part of the Facebook JavaScript SDK, which provides methods for user authentication and interaction with Facebook's social platform. The `LoginOptions` interface specifies various options for the login process, such as `auth_type`, `scope`, and `response_type`. However, there are situations where additional configuration parameters are required to customize the authentication process further.

The `extras` object is one such parameter, enabling developers to provide additional configuration settings to the Facebook JavaScript SDK, allowing more fine-grained control and customization. This change is essential for developers who need to pass specific parameters during the login process, as it enables the SDK to recognize and utilize these additional settings.

**Proposed Changes**

I have added the `extras` object to the `LoginOptions` interface within the `index.d.ts` file. The `extras` object is an optional parameter and allows developers to include custom configuration settings during the login process.

Here's how the `LoginOptions` interface looks after this change:

```typescript
interface LoginOptions {
    auth_type?: "reauthenticate" | "reauthorize" | "rerequest" | undefined;
    scope?: string | undefined;
    return_scopes?: boolean | undefined;
    enable_profile_selector?: boolean | undefined;
    profile_selector_ids?: string | undefined;
    config_id?: string | undefined;
    response_type?: string | undefined;
    override_default_response_type?: boolean | undefined;
    extras?: { [key: string]: any } | undefined;  // Added extras object definition
}

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
## Formatting

The following files are not formatted:
1. types/facebook-js-sdk/index.d.ts

Consider running `npx dprint fmt` on these files to make review easier.
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 63289fcf52a076c06d7b23b48f5e80fb31162bf0
</p>

**Comment by sandersn** [comment]
Looks like the bot got confused during the monorepo upgrade. This is still waiting on review from @amritk @mahmoudzohdi @fluidsonic @bengry

**Comment by gabritto** [comment]
@maina-david no owner has reviewed this PR, and as a DT maintainer with no context of this particular package, it's hard to review this.
Could you provide any evidence that this `extras` property is part of the original API and should be included in the types? This would make it possible for a DT maintainer to review this PR. I looked at https://developers.facebook.com/docs/javascript, but couldn't find any reference to this `extras` property. Thank you.

**Comment by SteniMariyaThomas** [comment]
https://developers.facebook.com/docs/whatsapp/embedded-signup/embed-the-flow/#step-7--optional---session-logging

@gabritto Here U can find the reference for extras

**Review by andrewbranch — APPROVED** [review]

**Pull request closed without merging**
